thread/common/omrthreadattr.c

```diff
@@ -33,7 +33,7 @@
 #include "omrthreadattr.h"
 
 #define J9THREAD_ATTR_IS_VALID(attr) ((attr) && (*(attr)) && ((*(attr))->size == sizeof(omrthread_attr)))
-#define J9THREAD_VALUE_OUT_OF_RANGE(val, lo, hi) (((val) < (lo)) || ((val) > (hi)))
+#define J9THREAD_VALUE_WITHIN_RANGE(val, lo, hi) (((val) >= (lo)) && ((val) <= (hi)))
```

Comment:
Two comments:
* this should put the extremes first and last (see a recent addition to the [coding standard](https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md)):
```c
#define J9THREAD_VALUE_WITHIN_RANGE(val, lo, hi) (((lo) <= (val)) && ((val) <= (hi)))
```
 
* won't this replace the 'always false' warning with an 'always true' warning (unsigned values are always >= 0)?

Reply:
The new changes should address the above concerns.